service/src/test/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/WorkspaceCloneUtilsTest.java

```diff
@@ -316,33 +311,4 @@ private static void assertControlledResourceCommonField(
     assertEquals(sourceResource.getAccessScope(), resourceToClone.getAccessScope());
     assertEquals(sourceResource.getAssignedUser(), resourceToClone.getAssignedUser());
   }
-
-  @Test
-  public void createDestinationResourceLineage_sourceLineageIsNull() {
```

Comment:
can we preserve the test in the wsmResource test?

Reply:
In PF-2196 I added a test to construct the resource object by two paths with null and non-null lineage. I think that covers the cases.